commands/bash/container/get-dashboard-url/get-dashboard-url

```diff
@@ -2,12 +2,12 @@
 set -eu -o pipefail
 
 function get_dashboard_url {
-    base=$(kubectl cluster-info | grep "Kubernetes master" | cut -d '/' -f3)
-    if [[ $CLUSTER_CODE = "minikube" ]]; then
+    if [[ $CLUSTER_CODE == "minikube" ]]; then
+        base=$(kubectl cluster-info | grep "Kubernetes master" | cut -d '/' -f3)
```

Comment:
wait, does this work? Should we change this output `please run "minikube dashboard" from your host` or something?

Reply:
well, interesting point ... it works, in that it gives some output:
```
$ aladdin --dev get-dashboard-url
Including environment variables from script /root/aladdin-config/minikube/env.sh
START ENVIRONMENT CONFIGURATION=============================================
CLUSTER_CODE = minikube
NAMESPACE = default
Context "default.minikube" created.
Switched to context "default.minikube".
END ENVIRONMENT CONFIGURATION===============================================
192.168.99.100:30000
```

But that URL is bogus ... so yeah lemme update it with a message